x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/transform/transforms/TransformConfigTests.java

```diff
@@ -511,7 +511,7 @@ public void testRewriteForUpdate() throws IOException {
         assertTrue(transformConfigRewritten.getSettings().getDatesAsEpochMillis());
         assertFalse(transformConfigRewritten.getSettings().getAlignCheckpoints());
 
-        assertWarnings(TransformDeprecations.ACTION_MAX_PAGE_SEARCH_SIZE_IS_DEPRECATED);
+        assertWarnings(org.apache.logging.log4j.Level.WARN, TransformDeprecations.ACTION_MAX_PAGE_SEARCH_SIZE_IS_DEPRECATED);
```

Comment:
Any particular reason these are fully qualified?

Reply:
Weird .. Intellij chose to do that.  Fixed in [`0718f87` (#80374)](https://github.com/elastic/elasticsearch/pull/80374/commits/0718f8716aaa8ff7b7075501e7b258887f4d3af3)